src/Qwiq.Core/Project.cs

```diff
@@ -48,11 +47,18 @@ public bool Equals(IProject other)
 
         public IWorkItemTypeCollection WorkItemTypes => _wits.Value;
 
+        public IQueryFolderCollection QueryHierarchy => _queryHierarchy.Value;
+
         public override bool Equals(object obj)
         {
             return ProjectComparer.Default.Equals(this, obj as IProject);
         }
 
+        public bool Equals(IProject other)
```

Comment:
Is there a test for this?

Reply:
There are integration tests which use equals, but no unit tests